tests/register.spec.coffee

```diff
@@ -11,22 +11,32 @@ describe 'Device Register:', ->
 
 	describe '.generateUUID()', ->
 
-		it 'should return a string', ->
-			uuid = register.generateUUID()
-			expect(uuid).to.be.a('string')
-
-		it 'should have a length of 62 (31 bytes)', ->
-			uuid = register.generateUUID()
-			expect(uuid).to.have.length(62)
-
-		it 'should generate different uuids each time', ->
-			uuid1 = register.generateUUID()
-			uuid2 = register.generateUUID()
-			uuid3 = register.generateUUID()
-
-			expect(uuid1).to.not.equal(uuid2)
-			expect(uuid2).to.not.equal(uuid3)
-			expect(uuid3).to.not.equal(uuid1)
+		it 'should return a string in its callback', (done) ->
```

Comment:
Can you use the `chai-as-promised` plugin for these three tests? See https://github.com/domenic/chai-as-promised/. There are a few examples on this spec.